plugin/lighthouse/parse_test.go

```diff
@@ -0,0 +1,109 @@
+package lighthouse
```

Comment:
NOT REVIEWED because I assume it's a copy-paste

Reply:
Test contents are copy paste but they're not using ginkgo, so changed accordingly.